### PR TITLE
Initial proposal for our RFC process

### PR DESCRIPTION
**Pull request by juwara0**
This change is [Reviewable](https://reviewable.io/reviews/ember-forge/ember-forge-ui/1)

### CODE REVIEW

File: rfcs/0000-template.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- ember-forge-ui Issue: (leave this empty)
+
+# Summary
+
+One para explanation of the feature.
```
Comment (sglanzer) [inline rfcs/0000-template.md]: para -> paragraph

File: rfcs/0000-template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
```
Comment (sglanzer) [inline rfcs/0000-template.md]: We should cite that this file is pulled directly from Ember's RFC template

Comment (sglanzer) [inline rfcs/0000-template.md]: Looks like this was handled below in the README, so that should be fine

**Comment by juwara0** [comment]
Review status: 0 of 7 files reviewed at latest revision, 2 unresolved discussions.

---

*[rfcs/0000-template.md, line 7 \[r1\]](https://reviewable.io:443/reviews/ember-forge/ember-forge-ui/1#-KEmp52MilSeQAaiRlt3-r1-7:-KEpmCFdjPZj9iWHI3GZ:-499146009) ([raw file](https://github.com/ember-forge/ember-forge-ui/blob/d7775575bb44b2d3e61e5c51686e461a72b363aa/rfcs/0000-template.md#L7)):*
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/ember-forge/ember-forge-ui/1)*

**Pull request merged**